doc/runtime_platform.rst

```diff
@@ -179,12 +179,12 @@ Context
 
     |comparable|
 
-.. function:: create_some_context(interactive=True, answers=None)
+.. autofunction:: create_some_context
 
     Create a :class:`Context` 'somehow'.
```

Comment:
Please also move this to the function docstring. :)

Reply:
:) What do you think of [9beccee](https://github.com/inducer/pyopencl/pull/732/commits/9beccee24e326438587c95ef6347966244866054)?